### PR TITLE
fix: support gpt-4

### PR DESCRIPTION
**Pull request by BlackStar1453**
GPT-4 is now functioning correctly, but it requires ensuring that your IP is located in East Asia (Japan, Singapore, South Korea, Taiwan, etc.). Otherwise, it is necessary to manually replace the server characters in the arkose_token with the region of your IP

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -274,6 +283,12 @@ devDependencies:
   rollup-plugin-visualizer:
     specifier: ^5.9.0
     version: 5.9.0
+<<<<<<< Updated upstream
```
Comment (maltoze) [inline pnpm-lock.yaml:286]: seems that conflicts were not resolved properly.

File: vite
Comment (maltoze) [inline vite:1]: please delete this file

Comment (BlackStar1453) [reply vite:1]: Which file? rollup-plugin-visualizer? I'll give it a try.

But  It seems the problem is that ofetch can't be installed via npm, I actually installed it by copying. Maybe I should upload this file?

Comment (maltoze) [reply vite:1]: You  committed a file named `vite`, and you don't need to install any new packages.

**Comment by maltoze** [comment]
1. Please do not install any packages by coping.
2. Should use `src/common/universal-fetch.ts` instead of `ofetch`.
3. Could we generate `arkose_token` on the client side?

**Comment by BlackStar1453** [comment]
It appears that ofetch can now be installed using npm.
I'm sorry, but I'm not sure how to utilize src/common/universal-fetch.ts instead of ofetch because it might be blocked by Cross-Origin Resource Sharing (CORS) restrictions.
I don't have the knowledge or understanding of that method. I looked at the code of ChatHub to imitate its implementation, but I don't know the specific implementation details.

**Comment by maltoze** [comment]
https://github.com/acheong08/ChatGPT/blob/85918ec0df394a2dd140b5436ee77e261af238b8/src/revChatGPT/V1.py#L157

**Pull request closed without merging**